### DISCUSSION TITLE
A Issues Regarding Translation into Simplified Chinese

### DISCUSSION THREAD

**Discussion by guanjiao23456** [ppy/osu#27640] [Ideas]
I apologize for the full text I will be writing in Chinese, although it seems out of line with the community rules, but it is because this article is written for translators. I will add an English translation at the end.
But even so, I hope more people see this article and maybe it will help community managers as well as game developers.

ranked在简体中文中被翻译为“上架”，繁体中文为“进榜”，显然，两边的译者对ranked有不同的理解。我认为应翻译为“可计分”。因为只有ranked的谱面和mod才能计算pp。上架更像是音乐被官方所认可，类似于商业音游中的收录，上架只适合描述谱面在谱面列表中的状态，不适合用于描述mod的状态。而进榜这一翻译虽然准确，但不太符合大陆地区语言习惯，故不推荐使用。
另外，如图所示。
Ranked is translated as " 上架" in simplified Chinese and " 进榜" in traditional Chinese, so obviously the translators on both sides of the border have different interpretations of ranked. In my opinion, it should be translated as " available for scoring". Because only ranked scores and mods can be counted for pp, ranked is more like the music being officially recognized, similar to the inclusion in a commercial rhythm game, and ranked is only suitable for describing the status of scores in beatmap listings, not for describing the status of mods. And the translation of the entry list, although accurate, is not quite in line with the language habits of mainland China, so it is not recommended to use it.
In addition, as shown in the picture.
![捕获](https://github.com/ppy/osu/assets/159292118/1d666fff-4c4d-4e32-a281-3e631c7b721c)

**Comment by cihe13375** [comment]
我个人认为ranked的翻译应该取决于场合：对于一张图的状态(ranked/qualified/loved/...)是“上架”，对于mod是(不)计入排名（“不计分”也可以，但有点微妙，因为unranked的分数也是会上传的，并且会算进total score和玩家等级，如果我没记错的话）。这基本上也是目前（stable和osu-web）的处理方法。

for the picture see https://github.com/ppy/osu/discussions/15452

**Comment by CloneWith** [comment]
The translation has been changed to `可进榜/不可进榜`.

I think we should blame this on the localization system, as it didn't provide enough information and context for translators to give a proper translation.

Again thanks for your question.
